tools/c7n_azure/c7n_azure/policy.py

```diff
@@ -331,3 +281,22 @@ def _is_subscribed_to_event(self, event, subscribed_events):
             return False
 
         return True
+
+    def _create_storage_queue(self, queue_name, session):
+        self.log.info("Creating storage queue")
+        storage_client = session.client('azure.mgmt.storage.StorageManagementClient')
+        storage_account = storage_client.storage_accounts.get_properties(
+            self.storage_account['resource_group_name'], self.storage_account['name'])
+
+        StorageUtilities.create_queue_from_storage_account(storage_account, queue_name)
```

Comment:
I am not entirely sure what gets returned from the `create_queue` call, but can we log some information like the Http Status if that's available?

Reply:
It just returns True. Silly, I know.

Reply:
Added try-catch-log-exit where we create the queue.